backend/src/main/java/com/rota/api/StaffController.java

```diff
@@ -123,4 +123,27 @@ private void verifyManagementRole(String authString) {
     verifyManagementRole(authString);
     return staffService.getActiveStaff();
   }
+
```

Comment:
After the backend PR is in (if it goes in current state) you no longer need to get the `AUTHORIZATION_HEADER`.

You can just call `verifyManagementRole()` and it will check the roles of the authorisation used in the current execution thread! 

Just a heads up of how it is currently working!